main.go

```diff
@@ -17,31 +17,38 @@ func glogUsage() {
 	os.Exit(2)
 }
 
-func initConfig() {
-	if err := config.Init(); err != nil {
-		glog.Fatal(err.Error())
-	}
-}
-
-func main() {
+func initLogging() {
 	// Check commandline options or "flags" for glog parameters
 	// to be picked up by the glog module
 	flag.Usage = glogUsage
 	flag.Parse()
 
+	if _, err := os.Stat(flag.Lookup("log_dir").Value.String()); os.IsNotExist(err) {
+		if (os.MkdirAll(flag.Lookup("log_dir").Value.String(), os.ModePerm) != nil) {
+			glog.Errorf("Failed to open custom log directory; defaulting to /tmp! Error: %v", flag.Lookup("log_dir").Value, err)
```

Comment:
You're logging an error here but you were successfully able to create the log_dir

Reply:
You mean change to Infof rather than Errorf right. 
I was thinking in the Dockerfile framework :) when I was working with it

Reply:
Ah, nevermind, I misread that if-condition